pkg/k8s/apps/interface.go

```diff
@@ -38,6 +39,8 @@ type App interface {
 	// TODO: remove after people move to CLI >= 1.14
 	RestoreOriginal() error
 
+	// GetCloned returns the cloned app from Kubernetes
+	GetCloned(ctx context.Context, c kubernetes.Interface) (App, error)
```

Comment:
The idea of this function is to get from Kubernetes the cloned resource. This is needed because of this https://github.com/okteto/okteto/pull/3368/files#r1093193476

Reply:
What does "clone" mean in this context? Is it the `DevClone` what we are getting? 

If that's the case, WDYT about renaming to `GetDevClone()`?

Reply:
Clone means the dev container. The one we clone from the original one.

`DevClone` creates the resource in memory, but not in k8s. `GetCloned` would retrieve that once it is in k8s.

If you think is clearer, I can change it